.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-        python: [ 2.7, 3.7 ]
+        python: [ 3.10.7, 3.11.2]
```

Comment:
If I recall we discussed adding versions of python that currently ship with Enterprise?

Reply:
as per the discussion, python 3.7 and Python 3.9 will be part of the matrix.